WTWSMS/common/dynasties/WtWSMS_vanilla_dynasties.txt

```diff
@@ -0,0 +1,115 @@
+#missing from sogdian.txt character
+1045074=
+{
+	name="Kashgar"
+	culture="sogdian"
+	religion="zoroastrian"
+}
+1045076=
+{
+	name="Charkliq"
+	culture="sogdian"
+	religion="manichean"
+}
+1045078=
+{
+	name="Cherchen"
+	culture="sogdian"
+	religion="manichean"
+}
+1045079=
+{
+	name="Qiuzi"
+	culture="sogdian"
+	religion="manichean"
+}
+1045075=
+{
+	name="Kang"
+	culture="sogdian"
+	religion="manichean"
+}
+1051143=
+{
+	name="Khuda"
+	culture="sogdian"
+	religion="zoroastrian"
+}
+1051144=
+{
+	name="Bukhar"
+	culture="sogdian"
+	religion="zoroastrian"
+	used_for_random=no
+}
+1051145=
+{
+	name="Afshin"
+	culture="sogdian"
+	religion="khurmazta"
+}
+1040024=
+{
+	name="Vijaya"
+	culture="sogdian"
+	religion="buddhist"
+}
+1040025=
+{
+	name="Amala"
+	culture="sogdian"
+	religion="buddhist"
+}
+1040026=
+{
+	name="Asu"
+	culture="sogdian"
+	religion="buddhist"
+}
+1040027=
+{
+	name="Phrom"
+	culture="sogdian"
+	religion="buddhist"
+}
+1040028=
+{
+	name="Hurod"
+	culture="sogdian"
+	religion="buddhist"
+}
+1040057=
+{
+	name="Kasyar"
+	culture="sogdian"
+	religion="buddhist"
+}
+1040058=
+{
+	name="Sarikol"
+	culture="sogdian"
+	religion="buddhist"
+}
+
+#missing from tocharian.txt characters
+1040052=
+{
+	name="Kuci"
+	culture="tocharian"
+	religion="buddhist"
+}
+
+#missing from panjabi.txt characters
+1042100=
+{
+	name="Karkota"
+	culture="panjabi"
+	religion="hindu"
+	coat_of_arms=
+	{
+		data=
+		{
+0 2 0 0 0 0 0 			}
+		religion="hindu"
+	}
+}
```

Comment:
I would rather add them to the vanilla file.

Reply:
Moved to 00_dynasties.txt